code_examples/sdk_examples/src/workshop/attester/generateKeypairs.ts

```diff
@@ -12,16 +12,13 @@ export function generateKeypairs(mnemonic = mnemonicGenerate()): {
 
   const assertionMethod = Kilt.Utils.Crypto.makeKeypairFromUri(mnemonic)
 
-  const capabilityDelegation = Kilt.Utils.Crypto.makeKeypairFromUri(mnemonic)
```

Comment:
@MarkusGarmeister This is needed: https://github.com/KILTprotocol/docs/actions/runs/9696806639/job/26759525902?pr=318

Reply:
I added the variable to the file